bgpd/bgp_routemap.c

```diff
@@ -5708,6 +5762,35 @@ DEFPY_YANG(
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFPY_YANG(
+	match_community_limit, match_community_limit_cmd,
+	"[no$no] match community-limit [(1-1024)$limit]",
+	NO_STR MATCH_STR "Match BGP community limit\n"
+		  "Community limit number\n")
+{
+	const char *xpath =
+		"./match-condition[condition='frr-bgp-route-map:match-community-limit']";
+	char xpath_value[XPATH_MAXLEN];
+	int idx_limit = 3;
+
+	nb_cli_enqueue_change(vty, xpath, NB_OP_CREATE, NULL);
+	snprintf(xpath_value, sizeof(xpath_value),
+		 "%s/rmap-match-condition/frr-bgp-route-map:community-limit", xpath);
+
+	if (no) {
+		nb_cli_enqueue_change(vty, xpath, NB_OP_DESTROY, NULL);
+		return nb_cli_apply_changes(vty, NULL);
+	}
+
+	if (argc < idx_limit) {
+		vty_out(vty, "%% BGP limit count not given.\n");
```

Comment:
Somehow the description is not as I would expect... What BGP limit?

Reply:
done